docs/RepoStructure.md

```diff
@@ -0,0 +1,72 @@
+<!--- master only -->
+> ![warning](images/warning.png) This document applies to the HEAD of the calico-docker source tree.
+>
+> View the calico-docker documentation for the latest release [here](https://github.com/projectcalico/calico-docker/blob/v0.9.0/README.md).
+<!--- else
+> You are viewing the calico-docker documentation for release **release**.
+<!--- end of master only -->
+
+# Calico Repositories
+
+This respository contains the following:
+
+- `calico/node`: the Docker image used to run the Calico Felix agent 
+  (responsible for IP routing and ACL programming) and a BGP agent (BIRD) 
+  for distributing routes between hosts.
+- `calicoctl`: the command line tool used for starting a `calico/node` container,
+  configuring Calico policy, adding and removing containers in a Calico network via 
+  orchestration tools, and managing certain network and diagnostic administration.
+-  Documentation for using Calico in a containerized environment.
+
+It pulls together a number of related repositories that provide subsets of
+function for running Calico in a containerized environment.  These related 
+respositories are listed below.
+
+## Calico Libraries
+
+ - [calico](https://github.com/projectcalico/calico): This repo container the
+   heart of Calico networking.  It contains the implementation of Felix, the 
+   process that interfaces with the Linux kernel to configure routes and ACLs 
+   that control network policy connectivity.  In the calico-docker
```

Comment:
"calico-docker containerized environments" isn't right. Don't we call this `calico/node` elsewhere?